#include <string>

// get current python error message
std::string get_py_error(void);